src/styles/review.css

```diff
@@ -0,0 +1,43 @@
+.review {
+  align-self: flex-start;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  width: 100%;
+  height: 16px;
+}
+
+.card__review {
+  margin-top: 16px;
+}
```

Comment:
.card__review should be in the card.css as this is a child of the card block, not the review.